tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -40,10 +40,7 @@
     "StampedPropertyResponsibleAttorney": "ResponsibleAttorney",
     "StampedPropertyTeamMembers": "TeamMembers",
     "StampedPropertyBlockedUploadUsers": "BlockedUploadUsers",
-    "StampedPropertyMatterCenterUserEmails": "MatterCenterUserEmails",
```

Comment:
These stamped properties should be removed as we are moving to dynamic hierarchy. These properties will be read from the content types section in the appsettings.json file